### ISSUE TITLE
Colorscheme Error after updating to 25.2.5

### ISSUE THREAD

**Issue by Crucialjun** [syncfusion/flutter-widgets#1870]
### Bug description

After updating to 25.2.5 the app is not building because of the following color scheme error 

Error: The getter 'surfaceContainerHighest' isn't defined for the class 'ColorScheme'

### Steps to reproduce

Update to version 25.2.5

### Code sample

<details open><summary>Code sample</summary>

```dart
[Add your code here]
```

</details>

### Screenshots or Video

<details open>
<summary>Screenshots / Video demonstration</summary>

[Upload media here]

</details>

### Stack Traces

../../../AppData/Local/Pub/Cache/hosted/pub.dev/syncfusion_flutter_gauges-25.2.5/lib/src/linear_gauge/axis/linear_axis_renderer.dart:42:29: Error: The getter 'surfaceContainerHighest' isn't defined for the class 'ColorScheme'.
 - 'ColorScheme' is from 'package:flutter/src/material/color_scheme.dart' ('/C:/flutter/packages/flutter/lib/src/material/color_scheme.dart').
Try correcting the name to the name of an existing getter, or defining a getter or field named 'surfaceContainerHighest'.
        ? theme.colorScheme.surfaceContainerHighest
                            ^^^^^^^^^^^^^^^^^^^^^^^
../../../AppData/Local/Pub/Cache/hosted/pub.dev/syncfusion_flutter_gauges-25.2.5/lib/src/linear_gauge/axis/linear_axis_renderer.dart:47:29: Error: The getter 'surfaceContainerHighest' isn't defined for the class 'ColorScheme'.
 - 'ColorScheme' is from 'package:flutter/src/material/color_scheme.dart' ('/C:/flutter/packages/flutter/lib/src/material/color_scheme.dart').
Try correcting the name to the name of an existing getter, or defining a getter or field named 'surfaceContainerHighest'.
        ? theme.colorScheme.surfaceContainerHighest
                            ^^^^^^^^^^^^^^^^^^^^^^^
../../../AppData/Local/Pub/Cache/hosted/pub.dev/syncfusion_flutter_gauges-25.2.5/lib/src/linear_gauge/axis/linear_axis_renderer.dart:127:29: Error: The getter 'surfaceContainerHighest' isn't defined for the class 'ColorScheme'.
 - 'ColorScheme' is from 'package:flutter/src/material/color_scheme.dart' ('/C:/flutter/packages/flutter/lib/src/material/color_scheme.dart').
Try correcting the name to the name of an existing getter, or defining a getter or field named 'surfaceContainerHighest'.
        ? theme.colorScheme.surfaceContainerHighest
                            ^^^^^^^^^^^^^^^^^^^^^^^
../../../AppData/Local/Pub/Cache/hosted/pub.dev/syncfusion_flutter_gauges-25.2.5/lib/src/linear_gauge/axis/linear_axis_renderer.dart:132:29: Error: The getter 'surfaceContainerHighest' isn't defined for the class 'ColorScheme'.
 - 'ColorScheme' is from 'package:flutter/src/material/color_scheme.dart' ('/C:/flutter/packages/flutter/lib/src/material/color_scheme.dart').
Try correcting the name to the name of an existing getter, or defining a getter or field named 'surfaceContainerHighest'.
        ? theme.colorScheme.surfaceContainerHighest
                            ^^^^^^^^^^^^^^^^^^^^^^^
../../../AppData/Local/Pub/Cache/hosted/pub.dev/syncfusion_flutter_gauges-25.2.5/lib/src/radial_gauge/axis/radial_axis_widget.dart:2395:40: Error: The getter 'surfaceContainerHighest' isn't defined for the class 'ColorScheme'.
 - 'ColorScheme' is from 'package:flutter/src/material/color_scheme.dart' ('/C:/flutter/packages/flutter/lib/src/material/color_scheme.dart').
Try correcting the name to the name of an existing getter, or defining a getter or field named 'surfaceContainerHighest'.
              ? _themeData.colorScheme.surfaceContainerHighest
                                       ^^^^^^^^^^^^^^^^^^^^^^^
Target kernel_snapshot failed: Exception

### On which target platforms have you observed this bug?

Android

### Flutter Doctor output

[√] Flutter (Channel stable, 3.19.5, on Microsoft Windows [Version 10.0.22623.1020], locale en-GB)
    • Flutter version 3.19.5 on channel stable at C:\flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 300451adae (8 weeks ago), 2024-03-27 21:54:07 -0500
    • Engine revision e76c956498
    • Dart version 3.3.3
    • DevTools version 2.31.1

[√] Windows Version (Installed version of Windows is version 10 or higher)

[√] Android toolchain - develop for Android devices (Android SDK version 34.0.0)
    • Android SDK at C:\Users\nicho\AppData\Local\Android\sdk
    • Platform android-34, build-tools 34.0.0
    • Java binary at: C:\Program Files\Android\Android Studio Preview2\jbr\bin\java
    • Java version OpenJDK Runtime Environment (build 17.0.10+0--11446219)
    • All Android licenses accepted.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files\Google\Chrome\Application\chrome.exe

[!] Visual Studio - develop Windows apps (Visual Studio Build Tools 2019 16.11.22)
    • Visual Studio at C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools
    • Visual Studio Build Tools 2019 version 16.11.33130.400
    • Windows 10 SDK version 10.0.19041.0
    X The current Visual Studio installation is incomplete.
      Please use Visual Studio Installer to complete the installation or reinstall Visual Studio.

[√] Android Studio (version 2023.2)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 17.0.9+0--11185874)

[√] Android Studio (version 2023.2)
    • Android Studio at C:\android-studio-2023.2.1.14-windows\android-studio
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 17.0.9+0--11084592)

[√] Android Studio (version 2023.3)
    • Android Studio at C:\Program Files\Android\Android Studio Preview2
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 17.0.10+0--11446219)

[√] VS Code (version 1.87.2)
    • VS Code at C:\Users\nicho\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension version 3.70.0

[√] VS Code (version 1.90.0-insider)
    • VS Code at C:\Users\nicho\AppData\Local\Programs\Microsoft VS Code Insiders
    • Flutter extension version 3.78.0

[√] Connected device (5 available)
    • SM A037F (mobile) • R9HRA0EWEVK        • android-arm64  • Android 13 (API 33)
    • Mi A2 (mobile)    • 192.168.0.32:38504 • android-arm64  • Android 12 (API 32)
    • Windows (desktop) • windows            • windows-x64    • Microsoft Windows [Version 10.0.22623.1020]
    • Chrome (web)      • chrome             • web-javascript • Google Chrome 124.0.6367.91
    • Edge (web)        • edge               • web-javascript • Microsoft Edge 125.0.2535.51

[√] Network resources
    • All expected network resources are available.

! Doctor found issues in 1 category.

**Comment by KingSajjad18676** [comment]
i think this update for support flutter 3.22 .... flutter 3.22 have so much changes and all libraries must update their codes :/

**Comment by iyar-avital** [comment]
I updated to flutter 3.22.0 and still have the same error

**Comment by selfprototype** [comment]
yep, same error

**Comment by LuHugo** [comment]
calendar have same error

**Comment by mahmudunnabiRain** [comment]
_syncfusion_flutter_datepicker-25.2.5/lib/src/date_picker/theme.dart_

![Screenshot 2024-05-21 at 11 18 44 PM](https://github.com/syncfusion/flutter-widgets/assets/52375732/dbb601cf-6aaa-4bce-bc7c-ce9e28124ec9)

`colorScheme.surfaceContainerHighest` is a InvalidType

for now I have change it to:
`Color? get weekNumberBackgroundColor => Colors.grey;`

Hoping that this will be fixed soon

**Comment by ac-arconomy** [comment]
+1

**Comment by nguyenxdat** [comment]
same here

**Comment by AndreBadenhorstAtCodingItDotIo** [comment]
I'm getting that the repo is discontinued. 
1 package is discontinued.

**Comment by AndreBadenhorstAtCodingItDotIo** [comment]
> Same here.
> 
> Downgrade version into 25.2.4 and wait new PR for fix it.

This worked for me, thank you!

**Comment by LokeshPalani** [comment]
Hi @Crucialjun,

We have updated our packages to be compatible with the latest Flutter SDK v3.22.0. In this version, Flutter introduced some breaking changes that were addressed in our packages. This may cause issues on your end if you are using a lower version of the Flutter SDK. So, we kindly request you to upgrade your Flutter SDK version to 3.22.0 or else use the **syncfusion_flutter_gauges** package version below [25.2.4](https://pub.dev/packages/syncfusion_flutter_gauges/versions/25.2.4).

If you have any further queries, please get back to us. We are always happy to assist you.

Regards,
Lokesh P.

**Comment by jlsimmo** [comment]
I have the same issue as above.  As of yesterday 21st May 2024 about 10am UK time I started getting the following 'unknown error' in Flutterflow when compiling my app that uses syncfusion calendar...

85 packages have newer versions incompatible with dependency constraints.
Try `flutter pub outdated` for more information.
Launching lib/main.dart on Web Server in debug mode...
Waiting for connection from debug service on Web Server...      
/root/.pub-cache/hosted/pub.dev/syncfusion_flutter_datepicker-25.2.5/lib/src/date_picker/theme.dart:83:55: Error: The getter 'surfaceContainerHighest' isn't defined for the class 'ColorScheme'.
 - 'ColorScheme' is from 'package:flutter/src/material/color_scheme.dart' ('/sdks/flutter/packages/flutter/lib/src/material/color_scheme.dart').
Try correcting the name to the name of an existing getter, or defining a getter or field named 'surfaceContainerHighest'.
  Color? get weekNumberBackgroundColor => colorScheme.surfaceContainerHighest;
                                                      ^^^^^^^^^^^^^^^^^^^^^^^
Waiting for connection from debug service on Web Server...         60.0s
Failed to compile application.

I'm stuck and I can't do anything.  I tried downgrading my dependencies to syncfusion_flutter_calendar: ^25.2.4 but still no luck.

Can anyone help me please?

Thanks,
John

**Comment by nguyenxdat** [comment]
@jlsimmo 

Remove pubspec.lock file, and change `syncfusion_flutter_calendar: ^25.2.4` to `syncfusion_flutter_calendar: 25.2.4`

**Comment by jlsimmo** [comment]
@nguyenxdat 

Thank you!

I searched my mac for all pubspec.lock files and deleted them all (none seem to be related to my project as they were all old with timestamps in December). 

I then changed all references in my FlutterFlow project to syncfusion_flutter_calendar: 25.2.4 - recompiled and got the same error. so then i downgraded to 25.2.3 and got the same issue... for some reason my FlutterFlow project still wants to include 25.2.5 - see below.

88 packages have newer versions incompatible with dependency constraints.
Try `flutter pub outdated` for more information.
Launching lib/main.dart on Web Server in debug mode...
Waiting for connection from debug service on Web Server...      
/root/.pub-cache/hosted/pub.dev/syncfusion_flutter_datepicker-25.2.5/lib/src/date_picker/theme.dart:83:55: Error: The getter 'surfaceContainerHighest' isn't defined for the class 'ColorScheme'.
 - 'ColorScheme' is from 'package:flutter/src/material/color_scheme.dart' ('/sdks/flutter/packages/flutter/lib/src/material/color_scheme.dart').
Try correcting the name to the name of an existing getter, or defining a getter or field named 'surfaceContainerHighest'.
  Color? get weekNumberBackgroundColor => colorScheme.surfaceContainerHighest;
                                                      ^^^^^^^^^^^^^^^^^^^^^^^
Waiting for connection from debug service on Web Server...         54.8s
Failed to compile application.

**Comment by jlsimmo** [comment]
No matter how i try to compile i just always get 25.2.5 - see below...

/root/.pub-cache/hosted/pub.dev/syncfusion_flutter_datepicker-25.2.5/lib/src/date_picker/theme.dart:83:55: Error: The getter 'surfaceContainerHighest' isn't defined for the class 'ColorScheme'.

I'm a bit of a FlutterFlow novice - I'm sure what I need to do is refresh/remove some sort of cache in FlutterFlow but I have no clue how to do it...

**Comment by jlsimmo** [comment]
@LokeshPalani  - do you have any ideas for me to the above problems please?

**Comment by nguyenxdat** [comment]
@jlsimmo 

I think FlutterFlow has cache with syncfusion_flutter_datepicker version 25.2.5.

You should `clean cache` before run `pub get` with version 25.2.4

**Comment by jlsimmo** [comment]
@nguyenxdat  - thanks for your help... I'm trying to clean cache and run pub get - but how/where do I do this? in Mac terminal?

**Comment by jlsimmo** [comment]
OK - but where is AppData\Local\Pub\Cache?

Is it a folder on my Mac? If so, I can't find it. It's not in my flutter directory.

**Comment by jlsimmo** [comment]
I'm still getting the same issue...

88 packages have newer versions incompatible with dependency constraints.
Try `flutter pub outdated` for more information.
Launching lib/main.dart on Web Server in debug mode...
Waiting for connection from debug service on Web Server...      
/root/.pub-cache/hosted/pub.dev/syncfusion_flutter_datepicker-25.2.5/lib/src/date_picker/theme.dart:83:55: Error: The getter 'surfaceContainerHighest' isn't defined for the class 'ColorScheme'.
 - 'ColorScheme' is from 'package:flutter/src/material/color_scheme.dart' ('/sdks/flutter/packages/flutter/lib/src/material/color_scheme.dart').
Try correcting the name to the name of an existing getter, or defining a getter or field named 'surfaceContainerHighest'.
  Color? get weekNumberBackgroundColor => colorScheme.surfaceContainerHighest;
                                                      ^^^^^^^^^^^^^^^^^^^^^^^
Waiting for connection from debug service on Web Server...         66.1s
Failed to compile application.

**Comment by jlsimmo** [comment]
I give up! There's no such folder on my Mac. 
I've spent 36 hours now, just trying to fix this issue - is there anyone out there from sync fusion that can help me please?

**Comment by jlsimmo** [comment]
I use Flutterflow - so I don’t really touch flutter directly JohnOn 22 May 2024, at 18:19, Yazad Homyar Dumasia ***@***.***> wrote:﻿
@jlsimmo Which flutter version are you using ?
For me I am using
Android Studio Giraffe | 2022.3.1 Patch 2
Flutter Version (Channel stable,  3.19.6)
Flutter plugins 77.1.1
Dart plugins 223.8977
Java SDK 17.0
environment:
sdk: '>=3.0.6 <4.0.0'
I perform this steps
First add my libraries
syncfusion_flutter_core: ^25.1.40
syncfusion_flutter_sliders: ^25.1.40
syncfusion_flutter_charts: ^25.1.40
syncfusion_flutter_xlsio: 25.1.40
then flutter pub get in android studio via Tools
then I manually update version value of above libraries in pubspec.lock file
then I go to AppData\Local\Pub\Cache\hosted and AppData\Local\Pub\Cache\hosted deleted folder and files which contain version 25.2.5.
And Then I run app and it will give normal warning regarding other packages some deprecated method and app working like normal

**Comment by jlsimmo** [comment]
OK - I found the pub.cache folder in the $home drive of my mac (it was hidden, cmd+sht+. unhides). I deleted the whole folder. I tried to test in FlutterFlow again... and got the same error...

87 packages have newer versions incompatible with dependency constraints.
Try `flutter pub outdated` for more information.
Launching lib/main.dart on Web Server in debug mode...
Waiting for connection from debug service on Web Server...      
/root/.pub-cache/hosted/pub.dev/syncfusion_flutter_datepicker-25.2.5/lib/src/date_picker/theme.dart:83:55: Error: The getter 'surfaceContainerHighest' isn't defined for the class 'ColorScheme'.
 - 'ColorScheme' is from 'package:flutter/src/material/color_scheme.dart' ('/sdks/flutter/packages/flutter/lib/src/material/color_scheme.dart').
Try correcting the name to the name of an existing getter, or defining a getter or field named 'surfaceContainerHighest'.
  Color? get weekNumberBackgroundColor => colorScheme.surfaceContainerHighest;
                                                      ^^^^^^^^^^^^^^^^^^^^^^^
Waiting for connection from debug service on Web Server...         64.2s
Failed to compile application.

**Comment by malik19995** [comment]
I am facing this issue on the latest version. 

I have Flutter SDK 3.22.0 
The same implementation worked on Syncfusion v23.2.7. 

I am using a SfCartesianChart with SplineAreaSeries. 
Unable to debug what's causing this. 

<img width="1018" alt="Screenshot 2024-05-24 at 7 59 10 AM" src="https://github.com/syncfusion/flutter-widgets/assets/46687299/d8ade8a4-06cc-4e2d-9def-0edde60033fb">
<img width="525" alt="Screenshot 2024-05-24 at 7 59 45 AM" src="https://github.com/syncfusion/flutter-widgets/assets/46687299/62c901e4-f392-47b8-8a40-c4eeb6f52c23">

**Comment by LokeshPalani** [comment]
Hi @jlsimmo | @YazadDuamasia | @nguyenxdat | @YazadDuamasia,

 
We would like to inform that the Flutter Flow using Flutter SDK v3.19.1. Our packages are now compatible with v3.22.0. To resolve the issue, we suggest you use the syncfusion_flutter_datepicker package version below [25.2.4](https://pub.dev/packages/syncfusion_flutter_datepicker/versions/25.2.4) without caret (^) sign to prevent upgrading the Flutter SDK to the latest version in the Flutter Flow.

 
Regards,
Lokesh P.

**Comment by felpfsf** [comment]
> Hi @Crucialjun | @selfprototype | @LuHugo | @mahmudunnabiRain | @nguyenxdat,| @iyar-avital | @KingSajjad18676 | @AndreBadenhorstAtCodingItDotIo,
> 
> We have updated our packages to be compatible with the latest Flutter SDK v3.22.0. In this version, Flutter introduced some breaking changes that were addressed in our packages. This may cause issues on your end if you are using a lower version of the Flutter SDK. So, we kindly request you to upgrade your Flutter SDK version to 3.22.0 or else use the **syncfusion_flutter_gauges** package version below [25.2.4](https://pub.dev/packages/syncfusion_flutter_gauges/versions/25.2.4).
> 
> We hope it will help you to resolve the issue. If you are still facing issue, we recommend that you check by clearing the cache using the below mentioned methods:
> 
> * ​In the Flutter terminal, add the **"flutter pub cache clean"** command to clear all the cache.
> * ​Close the running application and then add the **"flutter clean"** command.
> * ​Delete the **“pubspec.lock”** file.
> * ​Afterwards, add the **"flutter pub get"** command to get all the dependencies in the pubspec.yaml file.
> 
> Regards, Lokesh P.

I have the exact same error, tried to downgrade the syncfusion_flutter_calendar to 25.2.4 or .3 but the datepicker and core still installs with .5 version, so I had to downgrade them too on pubspec.yaml.

```
  syncfusion_flutter_calendar: 25.2.4
  syncfusion_flutter_datepicker: 25.2.4
  syncfusion_flutter_core: 25.2.4
```

**Comment by iimyzf** [comment]
I run into the same issue, I got over it by going to the **Source Control of GitHub** _(the little bar in the left of VSCode)_, and **Discard the changes of the pubspec.lock**.

Why did it work?? I really have no idea, because before doing that, I completely removed the .lock file and cleaned the cache and everything, but didn't work, but by just discarding the changes, it worked 👍

**Comment by LokeshPalani** [comment]
Hi @YazadDuamasia,

We have checked the reported issue that Syncfusion Flutter package version [25.2.5](https://pub.dev/packages/syncfusion_flutter_calendar/versions/25.2.5) is not compatible with the Flutter SDK version 3.10.6 to 3.19.1. To resolve the issue, we suggest you use the latest Flutter SDK version. Additionally, we have shared user guide documentation for the Syncfusion Compatibility Package version. Please find the compatible Syncfusion Flutter package version based on your Flutter SDK version below.

UG Link: https://help.syncfusion.com/flutter/system-requirements#sdk-version-compatibility

Regards,
Lokesh P.

**Comment by LokeshPalani** [comment]
Hi @YazadDuamasia,

We would like to inform you that 7 new [ColorScheme](https://api.flutter.dev/flutter/material/ColorScheme-class.html) were introduced in the Flutter SDK Version 3.22.0, and some properties have been deprecated. We have shared details below for 7 new color roles and deprecated color roles.

Please find the details for 7 new color roles included in the [ColorScheme](https://api.flutter.dev/flutter/material/ColorScheme-class.html):


- surfaceBright
- surfaceDim
- surfaceContainer
- surfaceContainerLow
- surfaceContainerLowest
- surfaceContainerHigh
- surfaceContainerHighest

| S No | Flutter SDK version 3.19.0 | Flutter SDK version 3.22.0 (After deprecated)|
|-----------------|-----------------|-----------------|
| 1  | background | surface  |
| 2  | onBackground  | onSurface |
| 3  | surfaceVariant  | surfaceContainerHighest  |

**Release Notes link for Flutter SDK Version 3.22.0:** https://docs.flutter.dev/release/release-notes/release-notes-3.22.0

**Breaking changes link for Flutter SDK Version 3.22.0:** https://docs.flutter.dev/release/breaking-changes#released-in-flutter-3-22

We hope that the above shared details will help you to know more about what is done in the Flutter SDK version 3.22.0. If you have any further queries, please feel free to reach out to us.

Regards,
Lokesh P.

**Comment by hira12333** [comment]
> Color? get weekNumberBackgroundColor => Colors.grey;

worked

**Comment by ergulecdemet** [comment]
I solved it by upgrading Flutter Sdk to 3.22.0.

**Comment by thisadee2897** [comment]
I solved it by upgrading Flutter Sdk to 3.22.2

**Comment by martipello** [comment]
using syncfusion_flutter_datepicker: ^26.1.39 with flutter 3.22.2 same error

Flutter 3.22.2 • channel stable • https://github.com/flutter/flutter.git
Framework • revision 761747bfc5 (3 weeks ago) • 2024-06-05 22:15:13 +0200
Engine • revision edd8546116
Tools • Dart 3.4.3 • DevTools 2.34.3

deleted all .lock files including environment ones still same error, worked around by using previous version 

  syncfusion_flutter_datepicker: 19.4.38

**Comment by LokeshPalani** [comment]
Hi @martipello,

We have validated your query. We recommend that you check by clearing the cache using the below mentioned methods:

- ​In the Flutter terminal, add the **"flutter pub cache clean"** command to clear all the cache.
- ​Close the running application and then add the **"flutter clean"** command.
- ​Delete the **“pubspec.lock”** file.
- ​Afterwards, add the **"flutter pub get"** command to get all the dependencies in the pubspec.yaml file.

If you are still facing issues after using above mentioned methods, we kindly request you to share the issue with a screenshot or screen recording. This will help us to assist you more effectively.

Additionally, we have shared user guide documentation for the Syncfusion Compatibility Package version. Please find the compatible Syncfusion Flutter package version based on your Flutter SDK version below.

UG Link: https://help.syncfusion.com/flutter/system-requirements#sdk-version-compatibility

Regards,
Lokesh P.

**Comment by martipello** [comment]
yes i already cleaned all caches and lock files and rebuilt the project, fix your library, its very easy to reproduce

**Comment by hwebservices** [comment]
the issue persists, no matter the version I use. I already followed the steps above too.

![image](https://github.com/syncfusion/flutter-widgets/assets/81170075/d2997d43-dc47-4870-aa76-a416f7a83f49)

**Comment by LokeshPalani** [comment]
Hi @martipello | @hwebservices, 


As mentioned in the previous update, the breaking changes were included in the Flutter SDK version **3.22.0**. To avoid this issue, we kindly request that you make sure that you are using the latest Flutter SDK version **3.22.2** for chart version [26.1.41+1](https://pub.dev/packages/syncfusion_flutter_charts/versions/26.1.41+1). Please let us know if you need any further assistance.


ColorScheme Breaking changes link: https://docs.flutter.dev/release/breaking-changes/new-color-scheme-roles

Regards,
Lokesh P.

**Comment by LokeshPalani** [comment]
Hi @martipello | @hwebservices,

We have updated our packages to be compatible with the Flutter SDK v3.22.0. In this version, Flutter introduced some breaking changes that were addressed in our packages. This may cause issues on your end if you are using a lower version of the Flutter SDK. To avoid this issue, we kindly request you to make sure that you are using the latest Flutter SDK version 3.22.2 for chart version [26.1.41+1](https://pub.dev/packages/syncfusion_flutter_charts/versions/26.1.41+1). Please let us know if you need any further assistance.

ColorScheme Breaking changes link: https://docs.flutter.dev/release/breaking-changes/new-color-scheme-roles#migration-guide

Regards,
Lokesh P.
